iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/VariantManagerImpl.java

```diff
@@ -137,11 +163,14 @@ public void cloneVariant(String sourceVariantId, List<String> targetVariantIds)
                 LOGGER.trace("Recycling variant array indexes {}", Arrays.toString(indexes));
             }
         }
-        if (extendedCount > 0) {
+        if (!overwritten.isEmpty()) {
```

Comment:
Ok but we could probably have merged the 2 blocks into only one, they are identical except for the log.

Reply:
I don't see how to merge the two blocks without making it less understandable

Reply:
Ok!